pkg/integrations/commands/list-permissions.go

```diff
@@ -0,0 +1,48 @@
+package commands
+
+import (
+	"fmt"
+
+	"github.com/gosuri/uitable"
+	"github.com/spf13/cobra"
+
+	"github.com/apono-io/apono-cli/pkg/aponoapi"
+)
+
+func ListPermissions() *cobra.Command {
+	var integrationID string
+	var resourceType string
+	cmd := &cobra.Command{
+		Use:     "permissions",
+		GroupID: Group.ID,
+		Short:   "List all permissions of integration resource type",
+		RunE: func(cmd *cobra.Command, args []string) error {
+			client, err := aponoapi.GetClient(cmd.Context())
+			if err != nil {
+				return err
+			}
+
+			resp, _, err := client.AccessRequestsApi.GetSelectablePermissions(cmd.Context(), integrationID, resourceType).Execute()
+			if err != nil {
+				return err
+			}
+
+			table := uitable.New()
+			table.AddRow("ID", "NAME")
+			for _, permission := range resp.Data {
+				table.AddRow(permission, permission)
```

Comment:
id and name are not always the same. maybe we should show only name?

Reply:
Moving forward we might change the API to return both so I prefer the output to look like this